src/pages/home/report/ReportActionCompose.js

```diff
@@ -11,6 +11,7 @@ import {withNavigationFocus} from '@react-navigation/compat';
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
 import {withOnyx} from 'react-native-onyx';
+import lodashIntersection from 'lodash/intersection';
```

Comment:
Not a huge deal, but I think we prefer using underscore over new lodash methods.

Reply:
I don't have any current PR related to any related module. I'll raise a separate PR to address this sometime tomorrow.